scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+#crm-container {
+  input[type=radio]:not(.old-radiocheckbox-style):checked+label[for]::after {
+    top: 7px;
+  }
+
+  button {
+    background: $brand-primary;
+    border: 0;
+    border-radius: $border-radius-base;
+    color: $crm-pure-white-color;
+    font-family: $font-family-base;
+    height: 36px;
+    line-height: 1;
+    padding: 11px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    &:hover {
+      background: darken($brand-primary, $crm-darken-percentage);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+    }
+
+    &.cancel {
+      background: $crm-white;
+      border: 1px solid $crm-copy;
+      color: $crm-cancel-color;
+
+      &:hover {
+        background: $crm-copy;
+        color: $crm-white;
+      }
+    }
+  }
+
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
+    border-top: 0;
+    margin-top: 0 !important;
+  }
+
+  #newLocation {
+    padding: 0 20px 20px 15px;
+
+    #crm-submit-buttons {
+      padding-left: 5px;
+    }
+
+    .crm-event-is_show_location br {
+      display: none;
+    }
+
+    h3 {
+      @include section-title;
+      margin: 0 -20px 20px -15px;
+    }
+  }
+
+  .crm-event-manage-location-form-block > table {
+    margin-left: 15px;
+
+    .status {
+      padding-left: 0;
+    }
+  }
+
+  #priceSet .message {
+    margin-right: 15px;
+  }
+
+  #contributionType .description {
+    padding-right: 15px;
+  }
+
+  fieldset#discount,
+  fieldset#map-field {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    > legend {
+      @include section-title;
+      display: block;
+      width: 100%;
+      margin-left: -20px;
+    }
+
+    table {
```

Comment:
we have forms and tables styled globally why this needs separate styling?

Reply:
Not sure that global styling is correct, please see before and after screens on fees section

Reply:
you need to be sure then, we cannot add unnecessary styles, *especially* if we add them for a specific page. tables (like any other generic element) should be styled globally.

If we need to deviate from the global style, we need to find out if the deviation is a common variation or if it's a one-time thing.

Reply:
@boriszaydlin you will need to find out that if we can use any pre-defined mixins or placeholders and then can use it here.

Reply:
We can not use any of existing mixins and there is no other page using such structure